lib/Basics/StaticStrings.cpp

```diff
@@ -369,6 +369,7 @@ std::string const StaticStrings::Spearhead("spearhead");
 std::string const StaticStrings::TargetConfig("targetConfig");
 std::string const StaticStrings::Term("term");
 std::string const StaticStrings::CommitIndex("commitIndex");
+std::string const StaticStrings::Details("details");
```

Comment:
please add new static strings as `std::string_view`. There is an example in this file already, afaik.